src/server/checkstyle.xml

```diff
@@ -83,10 +83,6 @@
           <property name="format" value="[\t]"/>
           <property name="message" value="Tabs in java file"/>
         </module>
-        <module name="RegexpSinglelineJava">
```

Comment:
Suggestion: Not sure this is intended to be here, but I personally find this checkstyle rule frustrating so I'm down with removing it if everyone else is.

Reply:
It is intended to be here (I talked with Alex before pushing it)

Reply:
Coolio.